Source/Charts/Charts/PieRadarChartViewBase.swift

```diff
@@ -573,11 +569,10 @@ open class PieRadarChartViewBase: ChartViewBase
     {
         if rotationEnabled && !rotationWithTwoFingers
         {
-            let touch = touches.first as NSUITouch!
-            
-            let touchLocation = touch?.location(in: self)
-            
-            processRotationGestureMoved(location: touchLocation!)
+            if let touch = touches.first {
```

Comment:
Can we move the `if let touch = touches.first` into `if rotationEnabled && !rotationWithTwoFingers`
i.e.
```
if rotationEnabled && !rotationWithTwoFingers, let touch = touches.first { ... }
```

Reply:
Actually, we don't need to check to see if `touches.first` is non-nil. We know there is at least one touch because the delegate method is called. Even Apple force unwraps this their sample code.

Reply:
done